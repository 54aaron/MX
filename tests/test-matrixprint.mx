int main() {
    Matrix m;

    m = [[4,3], [5,6], [3,2]];
    print_matrix(m);

    m = [[30,90,48], [20,30,18], [39,20,48], [67,38,92]];
    print_matrix(m);
    print_matrix([[0,1,0,3],[0,6,3,2],[4,5,5,3]]);
}