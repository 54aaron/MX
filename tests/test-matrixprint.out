[ 4, 3 ]
[ 5, 6 ]
[ 3, 2 ]

[ 30, 90, 48 ]
[ 20, 30, 18 ]
[ 39, 20, 48 ]
[ 67, 38, 92 ]

[ 0, 1, 0, 3 ]
[ 0, 6, 3, 2 ]
[ 4, 5, 5, 3 ]

